inc/targetbase.class.php

```diff
@@ -1038,6 +1038,94 @@ protected function parseTags($content, PluginFormcreatorFormAnswer $formanswer,
       return $content;
    }
 
+   protected function showConditionSettings($rand) {
+      global $DB;
+
+      echo '<tr class="line0">';
+      
+      echo '<td width="15%">' . __('Condition') . '</td>';
+      echo '<td width="45%">';
+
+      $options[0] = __('No');
+      $options[1] = __('Yes');
+
+      $params['value'] = $this->fields['has_condition'];
+      $params['on_change'] = 'change_condition()';
+      $params['width'] = "65px";
+      $params['rand'] = $rand;
+
+      Dropdown::showFromArray("has_condition", $options, $params);
+      $script = <<<JAVASCRIPT
+         function change_condition() {
+            $('#condition_question_title').hide();
+            $('#condition_question_value').hide();
+
+            switch($('#dropdown_has_condition$rand').val()) {
+               case  "0" :
+                  $('#condition_question_title').hide();
+                  $('#condition_question_question').hide();
+                  $('#condition_question_value').hide();
+                  break;
+               case "1":
+                  $('#condition_question_title').show();
+                  $('#condition_question_question').css("display", "inline");
+                  $('#condition_question_value').css("display", "inline");
+                  break;
+            }
+         }
+         change_condition();
+         function get_condition_value() {
+            var id = $("#dropdown_condition_question1").val();
+            $.get("conditionvalues.php?id="+id, function(data) {
+              $('#condition_question_value').html(data);
+            });
+         }
+JAVASCRIPT;
+      echo Html::scriptBlock($script);
+      echo '</td>';
+      echo '<td width="15%">';
+      echo '<span id="condition_question_title" style="display: none">' . __('Question', 'formcreator') . '</span>';
```

Comment:
maybe you can use PluginFormcreatorQuestion::getTypeName() to get a localized string here.